docs/releases/changelog.md

```diff
@@ -11,9 +11,37 @@ It pulls in the latest version of upstream libc++ and marks the beginning of
 
 !-->
 
+## libcu++ 1.6.0 (CUDA Toolkit 11.5)
+
+libcu++ 1.6.0 is a major release. It adds two new functions to the `cuda::std::barrier` API and
+uses `<nv/target>` as the primary dispatch mechanism for `cuda::std::atomic`. This is an ABI breaking release.
+
+Supported ABI Versions: 4 (default), 3, and 2.
+
+Included in: CUDA Toolkit 11.5.
+
+### New Features
+
+- #111: Adds two APIs to `cuda::std::barrier` for waiting on the parity of the barrier.
+  - Thanks to Olivier Giroux for this contribution.
```

Comment:
Technically speaking we don't need to call out Olivier as he's a core contributor, but it's fine. Always good to thank people.